autopart-hibernation.ks.in

```diff
@@ -0,0 +1,70 @@
+#test name: autopart-hibernation
+%ksappend repos/default.ks
+network --bootproto=dhcp
+
+%ksappend common/common_no_storage_and_payload.ks
+zerombr
+clearpart --all --initlabel
+# Include the generated autopart command.
+%include /tmp/ksinclude
+
+%packages
+%end
+
+%pre
+# Generate the autopart command.
+autopart="autopart --hibernation"
+echo "$autopart" >/tmp/ksinclude
```

Comment:
Why not adding `autopart --hibernation` directly to the kickstart file, avoiding `%include`?

Reply:
Seconded. The technique with generated includes is useful to know, but there is no reason to use it here. You have a known constant and take it on a ride through the system.